src/core/PaperScope.js

```diff
@@ -249,9 +249,9 @@ var PaperScope = Base.extend(/** @lends PaperScope# */{
      * Sets up an empty project for us. If a canvas is provided, it also creates
      * a {@link View} for it, both linked to this scope.
      *
-     * @param {HTMLCanvasElement|String} element the HTML canvas element this
+     * @param {HTMLCanvasElement|String|Size} element the HTML canvas element this
```

Comment:
@sasensi pretty sure this is over 80chars... does the hinter not complain?

Reply:
That's right, sorry.
About the hinter, if you talk about `gulp jshint` task, I think it has no rule about that.